lib/hsig.js

```diff
@@ -39,7 +39,7 @@ hsig.makeKey = function makeKey(priv) {
 };
 
 hsig.createKey = function createKey(pub) {
-  return sig0.createKey(sig0.algs.PRIVATEDNS, pub);
+  return sig0.createKey(sig0.algs.PRIVATEDNS, pub, 'secp256k1.');
```

Comment:
The algorithm name should be something along the lines of `BLAKE2BSECP256K1` since it uses blake2b in the signature hash.

Reply:
OK, updated.